include/zephyr/dt-bindings/clock/imx_ccm.h

```diff
@@ -66,4 +66,7 @@
 #define IMX_CCM_FLEXIO1_CLK		0x1100UL
 #define IMX_CCM_FLEXIO2_3_CLK		0x1101UL
 
+#define IMX_CCM_LPADC1_CLK 0x1200UL
```

Comment:
Please align indentation here (although if there is no SOC using the CCM driver that has an LPADC, this change can just be dropped)